Topics/User_Experience/Button_Design.md

```diff
@@ -0,0 +1,77 @@
+# **Button Design**
+
+## **Introduction**
+When creating software, 99.9% of the time, we will need at least one button for each page. Buttons are one of the most common interactions between users and software. However, many software developers underestimate how important buttons are in terms of user experience. A poorly designed button can be confusing, misleading, or even fraustrating enough so that users give up on using the softare. On this page, we will introduce multiple ways that a button can be designed, and their implication behind each design.
```

Comment:
Nice introduction! I think it would be better if you can describe one or two examples of poorly designed buttons so that readers can better understand how a poorly designed button is defined.

Reply:
Indeed! I will include an example image of a poorly designed button in my newest commit. Thank you for your suggestions!